### PR TITLE
Optional catch binding support

### PR DESCRIPTION
**Pull request by sanex3339**
Added optional catch binding support

### CODE REVIEW

**Comment by sanex3339** [comment]
@michaelficarra please check this PR

**Comment by sanex3339** [comment]
@michaelficarra also can you release a new version after the merge, please.

File: escodegen.js [hunk 1]
```diff
@@ -1176,15 +1176,19 @@
             withIndent(function () {
                 var guard;
 
-                result = [
-                    'catch' + space + '(',
-                    that.generateExpression(stmt.param, Precedence.Sequence, E_TTT),
-                    ')'
-                ];
+                if (!stmt.param) {
```
Comment (michaelficarra) [inline escodegen.js]: Always use a positive condition when there's an alternative branch. Drop the `!` and swap the branches.

**Comment by sanex3339** [comment]
@michaelficarra fixed

**Comment by sanex3339** [comment]
Thank you.

@michaelficarra can you release a new version?

**Comment by sanex3339** [comment]
Ping, i really need this release

**Comment by papandreou** [comment]
@sanex3339, you come off as quite rude with all those pings. Please be aware that this is a volunteer effort and that things take time. Why don't you publish a fork while you're waiting?

**Comment by sanex3339** [comment]
@michaelficarra please,release a new version with this fix

**Comment by michaelficarra** [comment]
@sanex3339 Published version 1.13.0.

**Comment by sanex3339** [comment]
Thank you!

**Pull request merged**